### DISCUSSION TITLE
drawing strictly horizontal or strictly vertical lines

### DISCUSSION THREAD

**Discussion by romange** [excalidraw/excalidraw#5485] [Q&A] [answered]
Hi, 
is it possible to make sure that the line I drew is horizontal (no angle deviations) ?

**Comment by sojusnik** [comment, marked as answer]
Try holding shift when drawing the line, but it's very buggy, f.i. sometimes it only works when editing a line that was already drawn. Maybe a dev can describe why?

**Reply by romange** [reply]
Thanks! It actually works for me!

**Reply by dwelle** [reply]
> Try holding shift when drawing the line, but it's very buggy, f.i. sometimes it only works when editing a line that was already drawn. Maybe a dev can describe why?

Cannot reproduce, works for me always 🤔

**Reply by dwelle** [reply]
Oh, with multi-point lines holding shift while adding a point (either during creation or afterwards in the line editor) doesn't work indeed. We should fix that.

**Reply by dwelle** [reply]
tracking issue https://github.com/excalidraw/excalidraw/issues/5255

**Reply by sojusnik** [reply]
> tracking issue #5255

Perfect, thanks!
